dashboard/models.py

```diff
@@ -803,7 +803,7 @@ class Session(db.Model):
             back_populates='sessions_reviewed')
     timepoint = db.relationship('Timepoint', uselist=False,
             back_populates='sessions')
-    scans = db.relationship('Scan', cascade='all, delete')
+    scans = db.relationship('Scan', cascade='all, delete', order_by="Scan.series")
```

Comment:
Yay! sorting by series number!

Reply:
Yeaaaah, I should have had that in the first place! Erin pointed it out to me yesterday.